indigo_api/views/documents.py

```diff
@@ -151,6 +153,42 @@ class AnnotationViewSet(DocumentResourceView, viewsets.ModelViewSet):
 
     def filter_queryset(self, queryset):
         return queryset.filter(document=self.document).all()
+    
+    def list(self, request, **kwargs):
```

Comment:
@longhotsummer here is the overriden `list()` method.

Reply:
looks good.